lib/main.dart

```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
     return ReorderableListView(
       onReorder: route.move,
       children: [
-        for (var c in route.places)
-          RouteCellWidget(c),
+        for (var c in route.places) RouteCellWidget(c),
       ],
     );
   }
 }
 
+class MyScaffold extends StatefulWidget{
+  @override
+  _MyScaffoldState createState() => _MyScaffoldState();
+}
+
+class _MyScaffoldState extends State<MyScaffold>{
+  bool expanded = true;
+  var rightPanelWidth;
+  var buttonWidth;
+  @override
```

Comment:
Please, keep blank lines between functions and variables (as well as any other class's members)